ModularBungalow/Kiriechem/reagent.dm

```diff
@@ -0,0 +1,67 @@
+//Kiriechem Chemicals
+/datum/reagent/medicine/bicardine //brings it back kinda
+	name = "Bicardine"
+	description = "A bruise reliever with a low overdose"
+	color = "#ECEC8D" // rgb: 236	236	141
+	taste_description = "spicy"
+	overdose_threshold = 6
+	reagent_state = SOLID
+
+/datum/reagent/medicine/bicardine/on_mob_life(mob/living/M)
+	M.adjustBruteLoss(-2*REM)
+	..()
+	return TRUE
+
+/datum/reagent/medicine/bicardine/overdose_process(mob/living/M)
+	M.adjustFireLoss(3*REM)
```

Comment:
so taking bicardine _and_ kelotane at the same time allows you to bypass the overdose, that's prolly not good

Reply:
Also by design, you have to FIND both and then take them.